nautobot/core/templates/components/panel/body_content_text.html

```diff
@@ -1,10 +1,8 @@
 {% load helpers %}
-{%  if render_as == 'json' %}
-    {{ text_content | render_json }}
-{% elif render_as == 'yaml' %}
-    {{ text_content | render_yaml }}
-{% elif render_as == 'markdown' %}
-    {{ text_content | render_markdown }}
-{% else %}
-    {{ text_content }}
+{% if render_as == 'json' %}{{ text_content | render_json }}
+{% elif render_as == 'json_pretty' %}<pre>{{ text_content | render_json }}</pre>
```

Comment:
I personally don't think we need to distinguish between these two cases - one `json` option that renders with `<pre>` tags included should be sufficient?

Reply:
Got it. I wasn't sure about needs in the project so originally I left those two options.